lib/email_tracking_retriever.py

```diff
@@ -115,7 +117,7 @@ def get_tracking(self, email_id, mail) -> Tracking:
     url = self.get_order_url_from_email(raw_email)
     price = self.get_price_from_email(raw_email)
     order_ids = self.get_order_ids_from_email(raw_email)
-    group = self.get_buying_group(raw_email)
+    (group, reconcile) = self.get_buying_group(raw_email)
```

Comment:
fyi: the parens in (group, reconcile) are unnecessary

Reply:
Done.